hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConfigKeys.java

```diff
@@ -675,6 +675,8 @@ public final class OzoneConfigKeys {
   public static final String HDDS_SCM_CLIENT_FAILOVER_MAX_RETRY =
       "hdds.scmclient.failover.max.retry";
 
+  public static final String OZONE_XCEIVER_METRICS_PERCENTILES_INTERVALS_SECONDS_KEY =
+      "ozone.xceiver.metrics.percentiles.intervals.seconds";
```

Comment:
@aierate suggest: `ozone.xceiver.client.metrics.percentiles.intervals.seconds`, how do you think?